### PR TITLE
[8.0][ADD] crm_timesheet_analytic_partner

### PR DESCRIPTION
**Pull request by JavierIniesta**
# 
# CRM Timesheet Analytic Partner

This module modifies phone call timesheets. It sets "Other partner" as partner
from phone call.
# Installation

This module is auto-installed by Odoo when _crm_timesheet_
and _analytic_partner_hr_timesheet_ are installed.
# Usage

In phone calls tree, if you add an analytic account and the call has any
duration and a partner, the created timesheet activity will have
"Other partner" assigned.

### CODE REVIEW

**Comment by elicoidal** [comment]
Please one module per PR

File: crm_timesheet_analytic_partner/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingenieria S.L. - Javier Iniesta
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "CRM Timesheet Analytic Partner",
+    "version": "8.0.1.0.0",
+    "images": [],
```
Comment (elicoidal) [inline crm_timesheet_analytic_partner/__openerp__.py]: Please remove empty keys

File: crm_timesheet_analytic_partner/tests/test_crm_phonecall.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# © 2016 Antiun Ingenieria S.L. - Javier Iniesta
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests.common import TransactionCase
+
+
+class CrmPhonecallCase(TransactionCase):
+    def test_timesheet_prepare(self):
+        analytic_account = self.env['account.analytic.account'].create({
+            'name': 'Cuenta prueba', 'type': 'normal', 'use_timesheets': True})
```
Comment (elicoidal) [inline crm_timesheet_analytic_partner/tests/test_crm_phonecall.py]: English please

**Comment by rafaelbn** [comment]
Runbot fails because:

```
le "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3139904-73-4b4491/openerp/addons/base/module/module.py", line 393, in state_update
    raise orm.except_orm(_('Error'), _("You try to install module '%s' that depends on module '%s'.\nBut the latter module is not available in your system.") % (module.name, dep.name,))
except_orm: ('Error', u"You try to install module 'crm_timesheet_analytic_partner' that depends on module 'analytic_partner_hr_timesheet'.\nBut the latter module is not available in your system.")
```

Please add dependency

cc @pedrobaeza

**Comment by pedrobaeza** [comment]
:+1: (code review)

Runbot is not working again, so I can't add the dependency. Tell me when runbot is up again and I'll add it.

**Comment by JavierIniesta** [comment]
Travis error fixed here: https://github.com/OCA/hr-timesheet/pull/74

File: crm_timesheet_analytic_partner/__openerp__.py (hunk 1 above)
Comment (JavierIniesta) [inline crm_timesheet_analytic_partner/__openerp__.py]: If I remove "images", Travis will fail.

Comment (pedrobaeza) [inline crm_timesheet_analytic_partner/__openerp__.py]: It's only a warning, but it's intended for creating snapshots that are intesively used on new apps.odoo.com

Comment (JavierIniesta) [inline crm_timesheet_analytic_partner/__openerp__.py]: Thanks! I added a screenshot.

**Comment by JavierIniesta** [comment]
Rebase done.

**Comment by JavierIniesta** [comment]
Runbot is up. Please add dependency.

Tanks!

**Comment by pedrobaeza** [comment]
Too late... runbot is again down. It seems to be a serious problem with the stability of the system. @gurneyalex, can you please check?

**Comment by rafaelbn** [comment]
This PR, fix this issue https://github.com/OCA/hr-timesheet/issues/62
To test and review this PR you only have to follow the steps of the issue.

@pedrobaeza could you try now to add dependency please? Still:

`File "/srv/openerp/instances/openerp-oca-runbot/parts/odoo-extra/runbot/static/build/3141251-73-fdae83/openerp/addons/base/module/module.py", line 393, in state_update
    raise orm.except_orm(_('Error'), _("You try to install module '%s' that depends on module '%s'.\nBut the latter module is not available in your system.") % (module.name, dep.name,))
except_orm: ('Error', u"You try to install module 'crm_timesheet_analytic_partner' that depends on module 'analytic_partner_hr_timesheet'.\nBut the latter module is not available in your system.")`

Thanks!

**Comment by rafaelbn** [comment]
Hi, I tested in local and this works fine :+1: ready to merge fix the bug! :smile:

**Comment by pedrobaeza** [comment]
:+1: 

I have added the dependency in runbot for the branch build to be green, so I merge.

**Pull request merged**